server/src/frontend_api.rs

```diff
@@ -568,10 +569,41 @@ security(
 )]
 #[post("/client/metrics")]
 async fn post_frontend_metrics(
+    req: HttpRequest,
     edge_token: EdgeToken,
+    connect_via: Data<ConnectVia>,
     metrics: Json<ClientMetrics>,
     metrics_cache: Data<MetricsCache>,
 ) -> EdgeResult<HttpResponse> {
+    if let Some(version) = req
+        .headers()
+        .get("unleash-sdk")
+        .and_then(|val| val.to_str().ok())
+        .map(str::to_owned)
+    {
+        crate::metrics::client_metrics::register_client_application(
+            edge_token.clone(),
+            &connect_via,
+            ClientApplication {
+                app_name: metrics.app_name.clone(),
+                environment: metrics.environment.clone(),
+                instance_id: metrics.instance_id.clone(),
+                connect_via: None,
+                connection_id: None,
+                interval: 15000,
+                started: Utc::now(),
+                strategies: vec![],
```

Comment:
It's not used

Reply:
Maybe use this PR as an excuse to drop strategies then. We used to use it to see which custom strategies a backend application had been initialised with. But with frontend, there are no strategies.

Reply:
So are you saying that we can fully remove it from the type since it's no longer used?

Reply:
I relaxed backend API constraint to make this one optional but to be backwards compat with older Unleash versions we probably need to keep sending empty strategies for now